server/models/user.js

```diff
@@ -1,11 +1,40 @@
 const mongoose = require('mongoose');
-
-const Schema = mongoose.Schema;
+const { Schema } = mongoose;
 
 const userSchema = new Schema({
-  // To Do: define user using mongoose.js
+  first_name: { type: String, required: true },
+  last_name: String,
+  password: { type: Number, required: true },
+  // Do we need reset ability yet?
+  resetToke: Number,
+  resetTokenExpiry: Date,
+  // To do: add validation
```

Comment:
I think we can do validation in the route that creates the users rather than on the model itself

Reply:
Great, that will probably be easier especially if we need to make updates to the type of validations.